bundle/edu.gemini.qv.plugin/src/main/scala/edu/gemini/qv/plugin/filter/core/FilterXMLFormatter.scala

```diff
@@ -28,9 +28,9 @@ object FilterXMLFormatter {
     <qvTool>
       <filters>{filters.map(format)}</filters>
       <axes>{axes.map(format)}</axes>
-      <charts>{histograms.map(_.toXml)}</charts>
+      <histograms>{histograms.map(_.toXml)}</histograms>
       <tables>{tables.map(_.toXml)}</tables>
-      <visCharts>{barCharts.map(_.toXml)}</visCharts>
```

Comment:
The renaming brings the node names in sync with the parser that tries to find those nodes in the xml.